src/Tribe/JSON_LD/Event.php

```diff
@@ -32,6 +32,21 @@ public static function instance( $name = null ) {
 	}
 
 	/**
+	 * Converts the start and end dates to the local timezone
+	 *
+	 * @param  string $event_tz_string
+	 * @param  string $date
+	 * @return srting
+	 */
+	public function local_timezone( $date, $event_tz_string ) {
```

Comment:
Going to suggest:

a) we rename this to `get_localized_iso8601_string()`
b) we change the access level from `public` to `private`

This will better reflect the job the method does and will let us kill it and remove it (without having to go through formal deprecation steps) once we're able to execute on some follow-up changes to the `tribe_get_start|end_date()` and `tribe_format_date()` functions (per convo).